rllib/algorithms/alpha_zero/mcts.py

```diff
@@ -84,14 +84,14 @@ def expand(self, child_priors):
     def get_child(self, action):
         if action not in self.children:
             self.env.set_state(self.state)
-            obs, reward, done, _ = self.env.step(action)
+            obs, reward, terminated, truncated, _ = self.env.step(action)
```

Comment:
Note for everything below: `done` usually gets replaced by `terminated, truncated`.